cartography/intel/aws/organizations.py

```diff
@@ -92,7 +92,7 @@ def load_aws_accounts(
 ) -> None:
     query = """
     MERGE (aa:AWSAccount{id: $ACCOUNT_ID})
-    ON CREATE SET aa.firstseen = timestamp()
+    ON CREATE SET aa.firstseen = timestamp(), aa.inscope=true
```

Comment:
We should maybe also be intentional to delete the `foreign` property if it exists.

Reply:
fixed in aws_foreign.accounts.json